doc/guides/dts/index.rst

```diff
@@ -712,6 +712,14 @@ generated in the same directory concatenating all board-related fixup files.
 The include file ``include/generated_dts_board.h`` includes both these generated
 files, giving Zephyr C source files access to the board's devicetree information.
 
+Since some boards may need special instructions to the device tree compiler
+(e.g. warning suppression) you can add extra flags in ``pre_dt_board.cmake``.
```

Comment:
We need to say where pre_dt_board.cmake is located.

Reply:
Updated